src/engine/Server.h

```diff
@@ -118,22 +120,47 @@ class Server {
   Awaitable<void> process(
       const ad_utility::httpUtils::HttpRequest auto& request, auto&& send);
 
-  /// Handle a http request that asks for the processing of a query.
+  /// Handle a http request that asks for the processing of an operation.
+  /// This is only a wrapper for `processQuery` and `processUpdate` which
+  /// does the error handling.
```

Comment:
I generally don't like the term `operation` in all of this as it is very generic. How about consistently (in all parameter and function names) calling it `queryOrUpdate` ? Not too long, but everyone knows what is meant here.